collector/runtime-benchmarks/hashmap/src/main.rs

```diff
@@ -0,0 +1,17 @@
+use benchlib;
+use benchlib::benchmark::BenchmarkSuite;
+
+fn main() {
+    let mut suite = BenchmarkSuite::new();
+
+    suite.register("hashmap-insert-10k", || {
+        let mut map =
+            hashbrown::HashMap::with_capacity_and_hasher(10000, fxhash::FxBuildHasher::default());
```

Comment:
I don't like this benchmark much, I'm hoping the suite will end up much less microbenchmark-y than this. But it's fine as a starting point.